.circleci/config.yml

```diff
@@ -238,7 +231,7 @@ jobs:
     steps:
       - checkout
       - *restore_cache_step
-      - run: tox -e 'bottle_contrib{,_autopatch}-{py27,py34,py35,py36}-bottle{11,12}-webtest' --result-json /tmp/bottle.results
+      - run: scripts/run-tox-scenario '^bottle_contrib\(_autopatch\)\?-'
```

Comment:
Good point. I actually have no clue why there's `TOX_SKIP_DIST=False` here. I just transformed the file. :)

Reply:
yeah, no worries, I do see these ones don't set the env.

It is mostly just to ensure that only do `python setup.py install` when we have to, since it is slow.


or at least that is the reasoning, I haven't tested this behavior/performance at all :/

Reply:
I'd prefer to keep this PR as is (simple transform) and tweak this in a different one later if that works for everyone.

Reply:
sure, if this isn't changing from what we have today, then fine by me.